### DISCUSSION TITLE
ERROR  When uploading the css the changes are not applied

### DISCUSSION THREAD

**Discussion by JonMP** [globaleaks/globaleaks-whistleblowing-software#3474] [Q&A]
[error globaleaks](https://github.com/globaleaks/GlobaLeaks/assets/29742036/4cc171f9-6e93-4fae-b58e-f3536dd9f97e)

Hello, after loading the css in the administration area and just reloading the page, it shows us the error that I show in the screenshot. how can I solve that? thank you

Refused to apply style from 'URL' because its MIME type ('application/octet-stream') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

**Comment by evilaliv3** [comment]
Thank you @JonMP 

We will check. Which version of the software are you using?

**Comment by JonMP** [comment]
good morning, the software version I'm using is 4.10.18. attached [capture]
![Version 4 10 18](https://github.com/globaleaks/GlobaLeaks/assets/29742036/eb5db7e7-c748-400a-8fe9-b1f3047a74e8)

**Comment by evilaliv3** [comment]
I see.

Thank you.

I think the bug is related to this specific version and we more recently
fixed.
Update to the current version and it should work

**Comment by JonMP** [comment]
thank you so much
